cmd/kyma/alpha/create/module/module.go

```diff
@@ -327,3 +422,87 @@ func (cmd *command) getRemote(nameMapping module.NameMapping) (*module.Remote, e
 
 	return res, nil
 }
+
+func (cmd *command) moduleDefinitionFromOptions() (*module.Definition, *Config, error) {
+
+	var def *module.Definition
+	var cnf *Config
+
+	nameMappingMode, err := module.ParseNameMapping(cmd.opts.NameMappingMode)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	if cmd.opts.WithModuleConfigFile() {
+		//new approach, config-file  based
+
+		moduleConfig, err := ParseConfig(cmd.opts.ModuleConfigFile)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		err = moduleConfig.Validate()
+		if err != nil {
+			return nil, nil, err
+		}
+
+		defaultCRPath, err := resolveFilePath(moduleConfig.DefaultCRPath, cmd.opts.Path)
```

Comment:
This DefaultCRPath should be optional, right? I got error without offer default CR path

Reply:
:+1: